_pages/who_uses_login.md

```diff
@@ -0,0 +1,37 @@
+---
+title: meta.who-uses-login.title
+description: meta.who-uses-login.description
+permalink: /who-uses-login/
+twitter_card: large
+image: /assets/img/login-gov-600x314.png
+---
+
+{% capture heading %}
+  {% t who-uses-login-page.hero.heading %}
+{% endcapture %}
+
+{% capture text %}
+  {% t who-uses-login-page.hero.text %}
+{% endcapture %}
+
+{% include hero.html class="who-uses-login" heading=heading text=text col_class="sm-col-6" %}
+
+<div class="bg-lightest-blue">
+  <div class="container who-uses-login">
+    <div class="partners bg-lightest-blue">
+      {{ site.translations[site.lang]["who-uses-login-page"]["partners"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+    </div>
+  </div>
+  <div class="bg-white">
+    <div class="container who-uses-login">
+      <div class="security">
+        {{ site.translations[site.lang]["who-uses-login-page"]["security"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+      </div>
+    </div>
+  </div>
+</div>
+
+{% capture banner_content %}
+  <p><a class="learn-account-creation link" href="{{ site.baseurl }}/create-an-account">{% t banner.one-account-for-govt.learn %}</a></p>
+{% endcapture %}
```

Comment:
From what I can infer from the designs, it appears that as far as the content of the partial, only the "Create an account" page would show anything _other_ than this "Learn more" link. Granted, it's used in only two places ("Home" and "Who Uses Login"), but it feels like something where the partial can either:

- Render a default value of the link in absence of an override
- Accept a boolean flag flipping between the "create" button and "learn more" link variations
- Otherwise determine context as to display link or button

Reply:
Paired with @aduth 🎉 

We are going to move forward with this as is for now and will re-visit to determine how to most effectively approach this.